src/assets/stylesheets/custom.scss

```diff
@@ -83,40 +83,28 @@ $textCenter: center;
 @include no-padding("all");
 
 //multiline truncate
-@mixin multiline-truncate($font-size: $blk-font-size, $line-height: $ttrunc-line-height, $lines-to-show: $trunc-lines, $trunc-bg: transparent) {
-  background: $trunc-bg;
-  display: block; /* Fallback for non-webkit */
-  display: -webkit-box;
-  max-height: $font-size*$line-height*$lines-to-show; /* Fallback for non-webkit */
-  font-size: $font-size;
-  line-height: $line-height;
-  -webkit-line-clamp: $lines-to-show;
-  -webkit-box-orient: vertical;
+@mixin multiline-truncate($line-height, $line-count, $bg-color){
   overflow: hidden;
-  text-overflow: ellipsis;
-  white-space: pre-wrap;
-  @-moz-document url-prefix() {
-    overflow: hidden;
-    position: relative;
-    &:before {
-      background: $trunc-bg;
-      bottom: 0;
-      position: absolute;
-      right: 0;
-      float: right;
-      content: '\2026';
-      margin-left: em(-36);
-      width: em(60);
-      padding-left: em(5);
-    }
-    &:after {
-      content: '';
-      background: $trunc-bg;
-      position: absolute;
-      height: em(50);
-      width: $width100;
-      z-index: 1;
-    }
+  position: relative;
+  line-height: $line-height;
+  max-height: $line-height * $line-count;
+  text-align: justify;
+  margin-right: -1em;
+  padding-right: 1em;
```

Comment:
padding-right is used to have space between words and ellipsis otherwise it will overlap.

margin-right is used to avoid unintentional space from right side of the block.

Image attached to explain 
<img width="222" alt="screen shot 2017-02-10 at 10 38 22 am" src="https://cloud.githubusercontent.com/assets/10396359/22814873/14436c0a-ef7d-11e6-8661-43cb8e7f5a3c.png">

<img width="59" alt="screen shot 2017-02-10 at 10 36 48 am" src="https://cloud.githubusercontent.com/assets/10396359/22814864/f4333526-ef7c-11e6-9161-a378ccfa99c1.png">